NeuroConn/preprocessing/preprocessing.py

```diff
@@ -43,7 +43,7 @@ def parse_path_windows_docker(path):
         path = '/' + path[0].lower() + '/' + path[2:]
     return path
 
-def parse_fmriprep_command(data_path, fmriprep_path, fs_license_path, work_path, participant_label, skip_bids_validation = True, nthreads, output_spaces, fs_recon_all, task, niprep_wrapper,mem_mb, sloppy = False, system = platform.system()):
+def parse_fmriprep_command(data_path, fmriprep_path, fs_license_path, work_path, participant_label, skip_bids_validation = True, nthreads, output_spaces, fs_recon_all, task, nipreps_wrapper,mem_mb, sloppy = False, system = platform.system()):
```

Comment:
I missed this one the first time, but the default arguments should never be followed by non-default arguments. place skip_bids before sloppy.